testgrid/config.yaml

```diff
@@ -2453,6 +2453,12 @@ test_groups:
 - name: pull-cluster-api-provider-openstack-make
   gcs_prefix: kubernetes-jenkins/pr-logs/directory/pull-cluster-api-provider-openstack-make
   num_columns_recent: 20
+- name: pull-cluster-api-provider-openstack-test
+  gcs_prefix: kubernetes-jenkins/pr-logs/directory/pull-cluster-api-provider-openstack-test
+  num_columns_recent: 20
+- name: pull-cluster-api-provider-openstack-check
+  gcs_prefix: kubernetes-jenkins/pr-logs/directory/pull-cluster-api-provider-openstack-check
+  num_columns_recent: 20
```

Comment:
you need to put the testgroup in one of the dashboard, probably https://github.com/kubernetes/test-infra/blob/3213b23d52905dca94623adcd41c7e24dbf30933/testgrid/config.yaml#L5804-L5807

Reply:
thanks again for your review! :+1: